src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -518,6 +518,37 @@ ExportQueryExecutionTrees::idToLiteral(const Index& index, Id id,
   }
 }
 
+// _____________________________________________________________________________
```

Comment:
Maybe it would be nice to have the optimization (as a parameter) to skip the encoded values (e.g. in some value getters like in yours we don't care about Ids etc.)